.github/workflows/build.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           cargo test --verbose --color always  -p lightning
           cargo test --verbose --color always  -p lightning-invoice
           cargo build --verbose  --color always -p lightning-persister
+          cargo build --verbose  --color always -p lightning-background-processor
+      - name: Test C Bindings Modifications on Rust ${{ matrix.toolchain }}
+        if: "! matrix.build-net-tokio"
+        run: |
+          RUSTFLAGS="--cfg=c_bindings" cargo test --verbose --color always  -p lightning
```

Comment:
Nit: extra space before “-p” and below

Reply:
Oops. Well its also the case in the upstream commit and I'm not gonna bother doing a PR to fix it there. :man_shrugging: